Rocket.Chat/Models/Mapping/MessageModelMapping.swift

```diff
@@ -29,10 +29,22 @@ extension Message: ModelMappeable {
             self.createdAt = Date.dateFromInterval(createdAt)
         }
 
+        if let createdAt = values["ts"].string {
+            let dateFormatter = DateFormatter()
+            dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+            self.createdAt = dateFormatter.date(from: createdAt)
```

Comment:
Let's add this process to `DateExtension`. This way we can reuse in other parts of the code.